server/src/modules/campaign-monitor/campaign-monitor.service.ts

```diff
@@ -112,12 +116,14 @@ export default class CampaignMonitorService {
 		}
 	}
 
-	public static async fetchNewsletterPreference(listId: string, email: string) {
+	public static async fetchNewsletterPreferences(
+		email: string
+	): Promise<Avo.Newsletter.Preferences> {
 		let url: string | null = null;
 		try {
-			url = `${
-				process.env.CAMPAIGN_MONITOR_SUBSCRIBERS_ENDPOINT as string
-			}/${listId}.json/?${queryString.stringify({ email })}`;
+			url = `${process.env.CAMPAIGN_MONITOR_SUBSCRIBERS_ENDPOINT}/${
+				process.env.CAMPAIGN_MONITOR_NEWSLETTER_OPTIN_LIST
+			}.json/?${queryString.stringify({ email })}`;
```

Comment:
I prefer this notation. But this is personal preference.
```suggestion
			url = queryString.stringifyUrl({
				url: `${process.env.CAMPAIGN_MONITOR_SUBSCRIBERS_ENDPOINT}/${process.env.CAMPAIGN_MONITOR_NEWSLETTER_OPTIN_LIST}.json`,
				query: { email },
			});
```

Reply:
Heb het nu overgenomen zoals het op de meeste andere plaatsen in de codebase is gebruikt :)